azure-pipelines.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       displayName: Clean Build
 
 - job: MacBuild
-  displayName: Mac Build
+  displayName: macOS builds
```

Comment:
Separate commit

Reply:
Fixed in ea38ae65, along with the other change you proposed to combine.